_docs/Kubernetes/Autoscaling.md

```diff
@@ -0,0 +1,155 @@
+---
+title: Autoscaling
+description: "How to use autoscaler: min and max count of nodes" 
+tags: [ featured, coretasks ]
+# permalink: /Autoscaling/
+---
+# Autoscaling: min and max count of nodes in Kubernetes cluster
+---
+
+In this task we will see how control count of nodes in Kubernetes cluster. When we indicate min node count, autoscaler will not scale-in count of nodes less then number of min node count. And when we indicate max node count, autoscaler will not scale-out count of nodes more then number of max node count.
+
+
+## Min node count
+
+- Go to Cloud page, choose Kubernetes and create your [Kubernetes cluster](https://ventuscloud.eu/docs/Kubernetes/Kubernetes%20Cluster) with existing parameters and add tag `min_node_count=3`:
+
+```
+Master count: 1
+Node count: 3
+Docker volume size (Gb): 50
+Node flavor: VC-2
+Master node flavor: VC-2
+```
+
+![](../../assets/img/autoscaling/min_node_count.png)   
+
+- Now connect to your cluster using [CLI](https://ventuscloud.eu/docs/Kubernetes/access-by-cli)  
+
+- Use command `kubectl get nodes` to make sure that we have minimum 3 nodes on our cluster:
+
+```
+NAME                         STATUS   ROLES    AGE     VERSION
+test-ptdzc7aqfngp-master-0   Ready    master   16m     v1.17.3
+test-ptdzc7aqfngp-node-0     Ready    <none>   14m     v1.17.3
+test-ptdzc7aqfngp-node-1     Ready    <none>   7m30s   v1.17.3
+test-ptdzc7aqfngp-node-2     Ready    <none>   82s     v1.17.3
+```
+Now we have 3 nodes on our cluster and autoscaler don't scale-in it lower then 3 nodes, as we indicated in min_node_count. 
+
+## Max node count
+
+- Go to Cloud page, choose Kubernetes and create your [Kubernetes cluster](https://ventuscloud.eu/docs/Kubernetes/Kubernetes%20Cluster) with existing parameters and add tag `max_node_count=1`:
+
+```
+Master count: 1
+Node count: 1
+Docker volume size (Gb): 50
+Node flavor: VC-2
+Master node flavor: VC-2
+```
+
+![](../../assets/img/autoscaling/max_node_count.png) 
+
+- Connect to your cluster using [CLI](https://ventuscloud.eu/docs/Kubernetes/access-by-cli)
+
+- Now to check how autoscaler will scale your nodes create `deployment-test.yaml` file with existing lines:    
```

Comment:
Now, to check how autoscaler will scale your nodes, create `deployment-test.yaml` file with the existing lines: